apps/site/pages/docs/getting-started/onboarding.mdx

```diff
@@ -51,15 +51,15 @@ Now that you have an overview of the app, let's start the project.
 2. In the **Setup & Create Project** page, ensure to define the basic configuration for your store:
 
 #### CMS Integration
-The option available to use a CMS in a FastStore project is [VTEX Headless CMS](/tbd), a VTEX'S Content Management solution.
+The option available to use a CMS in a FastStore project is [VTEX Headless CMS](https://www.faststore.dev/tutorials/cms/0), a VTEX'S Content Management solution.
 
 #### SEO Tags
 
 | Field | Description |
 | ------ | ----------- |
 | Title Tag | Type a keyword-optimized title for your store. For example: `Buy Online with the Best Game Store | GameStore.`. In this example, we used the length limit of around 50-60 characters to ensure it is fully displayed in search results without being truncated. It is also written in a way that accurately describes the webpage's content. |
 | Description Tag | Type a short description about your store, for example `Shop online at GameStore, the best game store for PC, Xbox, PS4 and Headphones. Explore our wide selection and buy your favorite games today!` |
-| Title Template | [TBD] For example `Headphones | GameStore`.|
+| Title Template | For example `Headphones | GameStore`.|
 | Site Author| type the name of the agency or person responsible for the storefront. |
 
 #### Github Acesss
```

Comment:
After the `Catalog` session, there's a section starting with `3.` that seems to be out of place. Should we correct it in this PR?

Reply:
Good! I removed the `3` and revised the sentence for conciseness.